teuthology/task/install/rpm.py

```diff
@@ -137,12 +137,12 @@ def _update_package_list_and_install(ctx, remote, rpm, config):
         log.info('Package version is %s', builder.version)
         builder.install_repo()
 
-    if install_packages:
-        log.info("Installing packages: {pkglist} on remote rpm {arch}".format(
-            pkglist=", ".join(rpm), arch=remote.arch))
-    else:
-        log.info("install_packages set to False: not actually installing packages")
-        return True
+    if not install_packages:
```

Comment:
Why changing the order of 'if'?

Reply:
@kshtsk Before this patch, no packages were installed if `install_packages` was False. This patch changes this so in the False case a single package "ceph-test" is installed. The `log.info("Installing packages: {pkglist} . . . .` is executed in both cases.

Reply:
BTW this will get refactored further when I backport https://github.com/ceph/teuthology/pull/1202